__tests__/app.test.js

```diff
@@ -113,3 +113,51 @@ describe("GET /api/reviews", () => {
       })
   });
 });
+
+describe("GET /api/reviews/:review_id", () => {
+  test("returns status code 200", () => {
+    return request(app)
+    .get("/api/reviews/1")
+    .expect(200);
+  })
+  test("responds with an object", () => {
+    return request(app)
+    .get("/api/reviews/1")
+    .then(({ body }) => {
+      expect(typeof body.review).toBe("object");
+    })
+  })
+  test("responds with object with appropriate properties", () => {
+    return request(app)
+    .get('/api/reviews/1')
+    .then(({ body }) => {
+      expect(body.review).toMatchObject({
+        review_id: expect.any(Number),
+        title: expect.any(String),
+        designer: expect.any(String),
+        owner: expect.any(String),
+        review_img_url: expect.any(String),
+        review_body: expect.any(String),
+        category: expect.any(String),
+        created_at: expect.any(String),
+        votes: expect.any(Number)
+      })
+    })
+  })
+  test("responds with 404 upon being a number out of range", () => {
```

Comment:
I'd like the test description to be more specific eg. responds with a 404 if given a valid but non-existent id